docs/neo3_guides/compiler_devpack/devpack.md

```diff
@@ -1,19 +1,46 @@
 # Devpack
 
+The neow3j devpack provides classes, methods and annotations required for writing smart contracts in
+Java. For example, if your smart contract needs to verify a transaction signature the devpack offers
+a method for that. Or, if you want to publish detailed information about the contract in its
+manifest, you can use one of the devpack's annotations.
+
+The following sections describe parts of the devpack's API and frequently used concepts and
+constructs.
+
 
 ## Neo Smart Contract API
 
-The Neo's smart contract API provides methods to retrieve blockchain information, access a contract's
-storage area, and interact with the execution environment in which a contract is run.
-The API is the same for all Neo developer packs, e.g.
+The biggest part of the devpack is the Neo smart contract API. It provides many functionalities, for
+example, to retrieve blockchain information, access a contract's storage area, and interact with the
+execution environment in which a contract is run. This API is the same for all Neo developer packs, e.g.
 [neo-boa](https://github.com/CityOfZion/neo-boa) or [neo-go](https://github.com/nspcc-dev/neo-go).
-You can find the documentation on it at the official 
+You can find the documentation on it int the official 
 [Neo docs](https://docs.neo.org/v3/docs/en-us/reference/scapi/fw/dotnet/neo.html). 
 
 Neow3j provides this API in the packages `io.neow3j.devpack.neo` and `io.neow3j.devpack.system`
 following the same naming and structure as described in the Neo docs.
 
 
+## Storage
+<!-- TODO: Document on how to use storage. -->
+Coming soon.
```

Comment:
I created this issue: https://github.com/neow3j/neow3j-docs/issues/6

That we don't forget. 👍